helix-view/Cargo.toml

```diff
@@ -27,6 +27,10 @@ helix-vcs = { version = "0.6", path = "../helix-vcs" }
 once_cell = "1.18"
 url = "2"
 
+strum = "0.25"
+strum_macros = "0.25"
+derive_more = "0.99"
```

Comment:
Agree with @the-mikedavis, the benefits here are dubious and this just inflates the code size, adding to build time with three compile time proc macro dependencies (that import yet more)